### ISSUE TITLE
Puppetboard v2.0.0 incompatible with latest Puppet Server 6.x and 5.x

### ISSUE THREAD

**Issue by sigv** [voxpupuli/puppetboard#553]
Puppetboard (as of v2.0.0) uses [v1 metrics](https://puppet.com/docs/puppetserver/6.3/metrics-api/v1/metrics_api.html) instead of [v2 (Jolokia) metrics](https://puppet.com/docs/puppetserver/6.3/metrics-api/v2/metrics_api.html). The v1 metrics API has been deprecated for a while and as of Puppet Server releases [6.9.1](https://puppet.com/docs/puppetserver/latest/release_notes.html#puppet-server-691) and [5.3.12](https://puppet.com/docs/puppetserver/5.3/release_notes.html#puppet-server-5312) (10 March 2020) these endpoints are disabled by default due to [CVE-2020-7943](https://puppet.com/security/cve/CVE-2020-7943). Trying to load pages from Puppetboard results in:

```text
ERROR:pypuppetdb.api: localhost:8080 over HTTP.
ERROR:puppetboard.utils:404 Client Error: Not Found for url: http://localhost:8080/metrics/v1/mbeans/puppetlabs.puppetdb.population%3Aname%3Dnum-nodes
```

**Comment by jvperrin** [comment]
From all I can tell from reading through the code this is a pypuppetdb issue (one of puppetboard's dependencies) instead of a puppetboard issue. When fetching metrics, [puppetboard is using `puppetdb.metric`](https://github.com/voxpupuli/puppetboard/blob/b67331a3e419d3c998beeae1552d731301bf9c6e/puppetboard/app.py#L116-L125) which is calling out to [this function in `pypuppetdb`](https://github.com/voxpupuli/pypuppetdb/blob/9eb9fb64c016bcee7c4cb52f458254afc84c9a9a/pypuppetdb/api.py#L887-L895) which uses [this v1 metrics mbean endpoint](https://github.com/voxpupuli/pypuppetdb/blob/9eb9fb64c016bcee7c4cb52f458254afc84c9a9a/pypuppetdb/api.py#L26), so I think this needs to be fixed on the pypuppetdb side of things and then the updated release there pulled into in this repo.

**Comment by sigv** [comment]
Thank you for the explanation! Just opened voxpupuli/pypuppetdb#172 for this.

**Comment by sigv** [comment]
This actually raises a further issue about Python 3 support, which could be considered #551.

Once the `pypuppetdb` update is released it can be expected to be `v2.1.0` or `v2.0.1` however this project is not compatible with these newer `v2.x` versions due to Python 3 incompatibility (33d8ac96f6411dba916737e8c1944feac3728229 followed by 3555936cdae18e07e594da08ed0165b22330b343).
